spec/endtoend/endtoend_spec.rb

```diff
@@ -144,15 +144,7 @@
       expect(@workspace.tsuru_command.stdout).to include query
     end
 
-    it "should be able to connect to the applitation via HTTPS with a valid cert" do
```

Comment:
Why did this move to the bottom? It should probably be next to the other HTTPS test, which it would replace, if anything.